logger.js

```diff
@@ -91,6 +91,12 @@ module.exports = (externalLogger, logFolder, moduleName = "unknownmodule")=> {
       if (externalLogger) {externalLogger.log("error", detail, table, moduleName);}
       if (validUiWindow()) {uiWindow.send("errorMessage", userFriendlyMessage || detail);}
     },
+    warning(detail, userFriendlyMessage, table) {
```

Comment:
I'm always confused with this method signature. If I have some json object as details and some string as event,  how is this going to be used? 

```javascript
logger.warning('file not found', JSON.stringify({filePath, operation, etc})); 
```

or 

```javascript
logger.warning(JSON.stringify({filePath, operation, etc}), 'file not found'); 
```

Reply:
@andrecardoso the second one is correct. detail is more technical, userFriendlyMessage is more 'user' oriented. Also detail goes to BQ, while userFriendly message not ( in the current implementation ).